### PR TITLE
make clearer what governance is used for at RW

### PR DESCRIPTION
**Pull request by shelbeeee**
I think it is important to make the distinction clearer between how onchain and offchain governance is used.

### CODE REVIEW

File: docs/community/03_governance_overview.md [hunk 1]
```diff
@@ -6,13 +6,21 @@ title: Overview
 import Grid, { Button } from '@site/src/components/Grid'
 
 Welcome to Radworks' version-controlled governance documentation! These docs consist of carefully curated information
-and resources that lay out the Radicle’s token voting governance process and how to participate in it.
+and resources that lay out the Radworks token voting governance process and how to participate in it.
 
-Radworks’ treasury is controlled by an on-chain governance mechanism ([Compound-Governor](https://wiki.tally.xyz/docs/compound-governor)) via the project’s native governance token ($RAD). Holders of this token can participate in the decision-making around the treasury by voting and creating on-chain governance proposals composed of executable actions. Currently, these actions include:
+The Radworks’ treasury is managed by an on-chain governance mechanism ([Compound-Governor](https://wiki.tally.xyz/docs/compound-governor)) via the project’s native governance token - $RAD. Holders of $RAD can participate in decision-making by voting on and creating Radworks Governance Proposals (RGPs). Voting power is derived directly from the number of tokens delegated to an address (1 token = 1 vote).
```
Comment (shelbeeee) [inline docs/community/03_governance_overview.md:11]: @abbey-titcomb  what do you think of adding 1 token = 1 vote at the end here?

File: docs/community/03_governance_overview.md [hunk 2]
```diff
@@ -34,8 +42,9 @@ Radworks’ treasury is controlled by an on-chain governance mechanism ([Compoun
   </Button>
 </Grid>
 
-### Contracts
+### Relevant Contracts
 
+* [Radworks Governance Contracts](https://github.com/radicle-foundation/radworks-governance/blob/main/manual.md#formal-review)
 * Governance: [0x690e775361AD66D1c4A25d89da9fCd639F5198eD](https://etherscan.io/address/0x690e775361AD66D1c4A25d89da9fCd639F5198eD)
```
Comment (shelbeeee) [inline docs/community/03_governance_overview.md:48]: Do we want to clarify what this actually linking to? Maybe "Governance Record"?

Comment (abbey-titcomb) [reply docs/community/03_governance_overview.md:48]: No, lets just leave it as is

File: docs/community/03_governance_overview.md (hunk 1 above)
Comment (abbey-titcomb) [reply docs/community/03_governance_overview.md]: Yes, good idea.

**Review by abbey-titcomb — APPROVED** [review]
lgtm

**Pull request merged**